addons/stock/data/stock_demo.xml

```diff
@@ -157,7 +170,7 @@
             <field name="company_id" eval="False"/>
         </record>
 
-        <record id="res_company_1" model="res.company">
+        <record id="res_company_1" model="res.company" context="{'default_code': 'CHIC1'}">
```

Comment:
with this https://github.com/odoo/odoo/pull/47201 and another small fix you could probably get rid of this hack and move it in the write bellow

Reply:
This comment is still accurate ?